numpy/core/code_generators/numpy_api.py

```diff
@@ -344,6 +344,8 @@
     # End 1.9 API
     'PyArray_CheckAnyScalarExact':          (300, NonNull(1)),
     # End 1.10 API
+    # End 1.11 API
```

Comment:
Should update this now that 1.12 is branched. The cversion and C-API number also need updating, see documentation in `doc/HOWTO_RELEASE.rst.txt`.